### PR TITLE
[`Refactor`] Coding Style, Updated and Applied `.editorconfig` Settings.

### PR DESCRIPTION
**Pull request by cschuchardt88**
# Change Log
- Added same `.editorconfig` settings from [`neo`](https://github.com/neo-project/neo/pull/3207) repo.
- Added missing from `neo` to this repo
- Applied `.editorconfig` errors and warning to projects.

Only Testing Done
---
image

@superboyiii @vncoelho  need your testing of every module.

@shargon @Jim8y Code review please.

### CODE REVIEW

**Review by vncoelho — COMMENTED** [review]
Hi Cris,

I am checking the code first.
There are too much changes in both PRs,here and in the PR on the core.

I think that the PR should be split in: basic coding changes, medium complex and other ones.

I will just take time testing after other Devs are in agreement with the styles and code itself, as well as the other changes that came with that.

**Comment by cschuchardt88** [comment]
@vncoelho mostly it's just changing to `var` variable type instead of `dotnet` types like `int`, `string`, etc. Should be easy and fast to scroll through.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,82 @@
+
+# Contributing to NEO
+Neo is an open-source project and it depends on its contributors and constant community feedback to implement the features required for a smart economy. You are more than welcome to join us in the development of Neo.
```
Comment (vncoelho) [inline CONTRIBUTING.md:3]: @cschuchardt88 ,organize the changes.
The PR is big enough by itself.

Divide in other PRs this file for example, the Security.md...etc...

After a complete organization I will review.

Break the changes. 
One first PR just with var...another with that...etc..

Comment (cschuchardt88) [reply CONTRIBUTING.md:3]: yes, but adding `.editorconfig` to the repo will block all `.github` workflows; having them fail for every PR. But I'll figure something out. But i still can't wait 5 years for a PR to get merged either...

**Comment by roman-khimov** [comment]
Maybe it'd be easier to do post-3.7, because my expectation is that we'll clean up PRs/issues here and then merge modules into the core. This will solve document/settings duplication immediately and then we'll just need to `dotnet format` the imported code.

**Comment by shargon** [comment]
Repository moved to [neo](https://github.com/neo-project/neo), please re-open there

**Pull request closed without merging**